test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -415,7 +524,17 @@ std::map<std::string, bool(*)()> const& GetSwitchMapping()
         { "VerifyToastPriority", &VerifyToastPriority },
         { "VerifyToastSuppressDisplay", &VerifyToastSuppressDisplay },
         { "VerifyToastExpiresOnReboot", &VerifyToastExpiresOnReboot },
-    };
+        { "VerifyUpdateToastProgressDataUsingValidTagAndValidGroup_Unpackaged", &VerifyUpdateToastProgressDataUsingValidTagAndValidGroup },
+        { "VerifyUpdateToastProgressDataUsingValidTagAndNullGroup_Unpackaged", &VerifyUpdateToastProgressDataUsingValidTagAndNullGroup },
+        { "VerifyUpdateNotificationDataWithNullTagAndValidGroup_Unpackaged", &VerifyUpdateNotificationDataWithNullTagAndValidGroup },
+        { "VerifyFailedUpdateNotificationDataWithNonExistentTagAndGroup_Unpackaged", &VerifyFailedUpdateNotificationDataWithNonExistentTagAndGroup },
+        { "VerifyFailedUpdateNotificationDataWithoutPostToast_Unpackaged", &VerifyFailedUpdateNotificationDataWithoutPostToast },
+        { "VerifyUpdateToastProgressDataUsingValidTagAndValidGroup", &VerifyUpdateToastProgressDataUsingValidTagAndValidGroup },
+         { "VerifyUpdateToastProgressDataUsingValidTagAndNullGroup", &VerifyUpdateToastProgressDataUsingValidTagAndNullGroup },
+        { "VerifyUpdateNotificationDataWithNullTagAndValidGroup", &VerifyUpdateNotificationDataWithNullTagAndValidGroup },
+        { "VerifyFailedUpdateNotificationDataWithNonExistentTagAndGroup", &VerifyFailedUpdateNotificationDataWithNonExistentTagAndGroup },
+        { "VerifyFailedUpdateNotificationDataWithoutPostToast", &VerifyFailedUpdateNotificationDataWithoutPostToast },
+      };
```

Comment:
@danielayala94 @pmpurifoy There is a piece of code in main() in this file:

 if (Test::AppModel::IsPackagedProcess())
    {
        auto activationInfo = winrt::ToastActivationInfo::CreateFromActivationGuid(c_toastComServerId);
        winrt::ToastNotificationManager::Default().RegisterActivator(activationInfo);
    }

Do you know why this change was introduced? It renders someof the unittests useless. Let me know. Thanks.